setup.cfg

```diff
@@ -7,4 +7,29 @@ default=db2+pyodbc://db2inst2:db2inst2@/test
 sqlite=sqlite:///:memory:
 
 [metadata]
+name = sqlalchemy-ibmi
 version = attr: sqlalchemy_ibmi.__version__
+description = SQLAlchemy support for Db2 on IBM i
+long_description = file: README.md
```

Comment:
Do we need to set `long_description_content_type` or does it automatically figure it out from the file extension?

Reply:
Have you seen this elsewhere? I believe the above is the best way to implement the long description. Followed example I saw on https://setuptools.readthedocs.io/en/latest/setuptools.html

Reply:
That example uses a file in ReStructured text, which is the default. `long_description_content_type` was added to allow for markdown.

See https://packaging.python.org/guides/making-a-pypi-friendly-readme/ and https://packaging.python.org/specifications/core-metadata/#description-content-type-optional

> If a Description-Content-Type is not specified, then applications should attempt to render it as text/x-rst; charset=UTF-8 and fall back to text/plain if it is not valid rst.

My question was whether setuptools has any magic to determine that from the file name or if it still defaults to rst.

Reply:
I think that it does need to be set, I think I misread your initial comment. Will add that to the config file.